plugins/copybuffer.zsh

```diff
@@ -0,0 +1,14 @@
+# copy the active line from the command line buffer 
+# onto the system clipboard (requires clipcopy plugin)
+
+copybuffer () {
+  if which clipcopy &>/dev/null; then
+    echo $BUFFER | clipcopy
+  else
+    echo "you must install clipcopy to use this keybinding"
```

Comment:
Since `clipcopy` is a function provided by OMZ, I would change this message to something more appropriate. For instance:
```
clipcopy function not found. Make sure you have Oh My Zsh correctly installed.
```